javascript/javascript-playground-questions/js-sum-items-in-object/js-sum-items-in-object.md

```diff
@@ -25,7 +25,10 @@ setupCode:
 
 ## Content
 
-Can you write a program that will take an object with at least five values as input, sum all of the values, and output the result?
+> 👩‍💻 Your task is to write a program that:
+> - **takes an object with at least five values as input**
```

Comment:
Afaik, you can refer to only values without mentioning keys?

Reply:
I think you can, but it can definitely get confusing

for example, how many values does this object have?

```js
const foo = {
  bar: ['a', 'b', 'c'],
  baz: '123',
  barbaz: {
    foo: ['1', '2']
  }
}
```